### DISCUSSION TITLE
Key & Salt im Plugin oder SIG in the URL

### DISCUSSION THREAD

**Discussion by csaeum** [FriendsOfShopware/FroshPlatformThumbnailProcessor#56] [Q&A] [answered]
Hello, I have the imgproxy running via Docker and Traefik.

When I had not yet configured the key and the salt, I was able to transfer the images from the SW6 and these were then also displayed in the shop with the URL from the imgproxy.
But when I passed the console command, it came out that the images were not generated but were always displayed under "Skipped".

I guess because he couldn't "login" to the imgproxy.

I have now set up key and salt according to these instructions:
https://www.digitalocean.com/community/tutorials/how-to-serve-next-generation-images-with-imgproxy-using-docker

I thought I'd generate a signature and put it in the URL to the imgproxy in the plugin.

But according to these instructions, you would always have to pass the key and SALT with each image call.

Am I seeing this correctly or am I doing something wrong?

**Comment by tinect** [comment, marked as answer]
Hi,

This plugin is not connecingt to anything others than shopware. :-) the browser of user will call urls you specified.

You will need a addon which will change the url further:
https://github.com/FriendsOfShopware/FroshPlatformThumbnailProcessorImgProxy (not tested for 2 years)

**Reply by csaeum** [reply]
Sorry I only saw it through further research that there is already a plugin that integrates the KEY and SALT with these.

https://github.com/FriendsOfShopware/FroshPlatformThumbnailProcessorImgProxy

could you adjust that? Because when installing, he tells me that something doesn't work right away with the URL's and he can't install the proxy plugin, so to speak?

**Reply by csaeum** [reply]
> Hi,
> 
> This plugin is not connecingt to anything others than shopware. :-) the browser of user will call urls you specified.
> 
> You will need a addon which will change the url further: https://github.com/FriendsOfShopware/FroshPlatformThumbnailProcessorImgProxy (not tested for 2 years)

Sorry, just saw your reply in the mail and skipped it earlier when replying.

Yes, I saw that and created an issue for the imgProxy addon. Because it doesn't work there because of the "incompatibility" with the install
